pinot-controller/src/main/resources/app/components/Query/VisualizeQueryStageStats.tsx

```diff
@@ -133,8 +133,7 @@ const generateFlowElements = (stats) => {
   const edges: Edge[] = [];
 
   const createFlowNode = (data, level, index, parentId) => {
-
-    const id = `${level}-${index}`; // Unique ID for the node
+    const id = `${parentId ? parentId + '_' : ''}${level}-${index}`; // Unique ID for the node
```

Comment:
This a bit strange. A node that is spooled should have different parent ids, right?

Reply:
After the changed strategy, this no longer applies. Parent information is already included as part of each nodeId itself.